epregressions/diffs/ci_compare_script.py

```diff
@@ -71,7 +71,14 @@ def main_function(file_name, base_dir, mod_dir, base_sha, mod_sha, make_public,
     runner = SuiteRunner(run_config, [])
 
     cleanup(mod_dir)
-    entry = runner.process_diffs_for_one_case(initial_entry, ci_mode=True)  # returns an updated entry
+    entry, message = runner.process_diffs_for_one_case(
+        initial_entry,
+        runner.build_tree_a,
+        runner.build_tree_b,
+        runner.test_output_dir,
+        runner.thresh_dict_file,
+        ci_mode=True
+    )  # returns an updated entry
```

Comment:
`process_diffs_for_one_case` is now a `@staticmethod` to make it easier to use with the process pool.  This required some extra parameters to be passed into it.